doc/classes/Object.xml

```diff
@@ -139,7 +139,7 @@
 			<argument index="4" name="flags" type="int" default="0">
 			</argument>
 			<description>
-				Connects a [code]signal[/code] to a [code]method[/code] on a [code]target[/code] object. Pass optional [code]binds[/code] to the call as an [Array] of parameters. Use [code]flags[/code] to set deferred or one-shot connections. See [enum ConnectFlags] constants.
+				Connects a [code]signal[/code] to a [code]method[/code] on a [code]target[/code] object. Pass optional [code]binds[/code] to the call as an [Array] of parameters. These parameters will be passed after any parameter passed call to [method emit_signal]. Use [code]flags[/code] to set deferred or one-shot connections. See [enum ConnectFlags] constants.
```

Comment:
I would rephrase to: "These parameters will be passed to the method after any parameter used in the call to [method emit_signal]."

Even then, it's not immediately obvious what this means, so I'd add an example of connecting a signal with binds + emitting it with a custom parameter, and showing what the signature of the callback should be like.

Reply:
CC @NathanLovato

Reply:
Yes you're right, it makes sense to provide an example. I'll make one and also rephrase as you proposed